protos/utils.proto

```diff
@@ -36,11 +36,11 @@ enum SIPrefix {
 // # Prefixed
 // A quantity annotated with an `SIPrefix`
 message Prefixed {
-  SIPrefix prefix = 1;  // The metric `SIPrefix`
-  oneof number {        // The numeric suffix, either as int, float, or string
-    int64 integer = 2;  // Int
-    double double = 3;  // Float
-    string string = 4;  // String
+  SIPrefix prefix = 1;          // The metric `SIPrefix`
+  oneof number {                // The numeric suffix, either as int, float, or string
+    int64 int64_value = 2;
+    double double_value = 3;
```

Comment:
Long discussed: 
Is this the time to dump the floating-point variant here? 
It's kinda antithetical to the point of the thing.

Reply:
We can discuss that at the next meeting. I will remove it and send the PR if we decide to. Prefer not to overload this change.